static/src/stylesheets/module/content/_media-player.scss

```diff
@@ -125,11 +131,12 @@ $ima-controls-height: 70px;
         &:after {
             content: '';
             position: absolute;
-            left: 1.85em;
-            top: 1.85em;
+            top: 50%;
+            left: 50%;
+            transform: translate3d(-40%, -50%, 0);
```

Comment:
do we need to add browser prefixed versions too? just because there's a `-moz-transform` on line 140 below